README.md

```diff
@@ -1,15 +1,17 @@
 # CTL Library
 
 This is a temporary library for collecting flow-of-control (think Haskell's `Control` modules) code.
-The plan is to eventually split this up into separate libraries. Currently, among other things, this includes:
+The plan is to eventually split this up into separate libraries. Currently, among other things, this
+includes:
 
 * Contravariant
 * Arrow
 * Monad Transformers (Reader, Writer, State)
 * Recursion Schemes
 
-**Expect breaking changes**
+## Expect breaking changes
```

Comment:
I think it was meant to be in bold.

Reply:
got a linter error, probably doesn't matter. there is currently no content in the section anyway.

Reply:
Added a . to avoid lint error.